apacheconfig/reader.py

```diff
@@ -21,5 +22,5 @@ def isdir(self, filepath):
     def listdir(self, filepath):
         return self._os.listdir(filepath)
 
-    def open(self, filename, mode='r', bufsize=-1):
-        return open(filename, mode, bufsize)
+    def open(self, filename, mode='r', encoding='utf-8', bufsize=-1):
```

Comment:
What's our policy concerning text input from now on? Is it that we always require `unicode` on Py2 and `str` on Py3? May be worth clarifying that, especially in `loads()` doctoring/signature?

Perhaps the same question for `dumps`.

Reply:
Yes, that's correct. I'll change the documentation to reflect this!

Reply:
In the documentation, let's use "unicode string" and "Text" to refer to these text types (unicode in 2 and str in 3)? `Text` because, if we'd like to use static typing in the future, `Text` is how mypy refers to unicode in python 2 / str in python 3. Thoughts?

Reply:
Sounds good to me!